Makefile

```diff
@@ -67,9 +67,7 @@ endif
 ocaml_checks: ocaml_version ocaml_word_size check_opam_switch
 
 libp2p_helper:
-ifeq (, $(MINA_LIBP2P_HELPER_PATH))
```

Comment:
Oh, apologies, I was testing my local build without nix and this env is not set by default, so I removed it temporarily and forgot to revert it afterwards

Reply:
Did you have any trouble working outside nix because of this env variable absence?

(I assume there shouldn't have been any trouble, but let me know)